web/modules/custom/collection/collection.behat.inc

```diff
@@ -351,6 +352,17 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
       $memberships = Og::membershipStorage()->loadMultiple($membership_ids);
       Og::membershipStorage()->delete($memberships);
 
+      // Some more workarounds for og_menu.
+      $storage = \Drupal::getContainer()->get('entity.manager')->getStorage('ogmenu_instance');
+      $values = [
+        OgGroupAudienceHelper::DEFAULT_FIELD => $collection->id(),
+      ];
```

Comment:
I suppose this workaround should also be fixed when OG cleans up its orphans. I guess the menus are also group content?